### ISSUE TITLE
aiohttp.ClientResponse.**mimetype** property

### ISSUE THREAD

**Issue by moden-py** [aio-libs/aiohttp#1349]
## Long story short

It supposed to be a popular use case to get the content's `mime type`.
Even `aiohttp` uses the next construction several times(though not always for response):

``` python
ctype = self.headers.get(hdrs.CONTENT_TYPE, '').lower()
mtype, stype, _, params = helpers.parse_mimetype(ctype)
```

Lets add an easy way to get `mime type`?
`aiohttp.helpers.parse_mimetype` already done.
## Expected behaviour

``` code
resp = await client_session.get(url)
async with resp:
    mtype, stype, suffix, params = resp.mimetype
```
## PS

If agree I will make appropriate PR.

**Comment by asvetlov** [comment]
`aiohttp.web.Request` has [content_type](http://aiohttp.readthedocs.io/en/stable/web_reference.html#aiohttp.web.Request.content_type) and `charset` properties.
I suggest adding the same to `aiohttp.ClientResponse`.

**Comment by asvetlov** [comment]
Fixed by #1352
